iothub/service/src/Feedback/MessageFeedbackProcessorClient.cs

```diff
@@ -111,7 +111,7 @@ public virtual async Task OpenAsync(CancellationToken cancellationToken = defaul
             {
                 if (MessageFeedbackProcessor == null)
                 {
-                    throw new Exception("Callback for message feedback must be set before opening the connection.");
+                    throw new InvalidOperationException("Callback for message feedback must be set before opening the connection.");
```

Comment:
Do we need to call out `InvalidOperationException` to be expected within the doc comments above?

Reply:
good point, sure